src/renderer/components/animate/animate.scss

```
@@ -21,51 +21,51 @@
 
 // Animations
 
-@mixin animate-opacity($enterDuration: 100ms, $leaveDuration: 150ms) {
+@mixin animate-opacity() {
```

Why removed from args?

```suggestion
@mixin animate-opacity($enterDuration: var(--enter-duration), $leaveDuration: var(--leave-duration)) {
```